{"uid":"c3d6141023eb8bce","name":"TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation","fullName":"tests.test_sign_in.TestFailedLogin#test_03_02_01_to_03_02_10_login","historyId":"13d92d3fc34ad3cd8390ae027b651bbc","time":{"start":1737857746507,"stop":1737857752024,"duration":5517},"description":"Check Success and Failed Login, no error validation","descriptionHtml":"<p>Check Success and Failed Login, no error validation</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560c0737c7ca <unknown>\n#1 0x560c06e742f0 <unknown>\n#2 0x560c06ec4035 <unknown>\n#3 0x560c06ec4251 <unknown>\n#4 0x560c06f0a054 <unknown>\n#5 0x560c06ee89dd <unknown>\n#6 0x560c06f073b3 <unknown>\n#7 0x560c06ee8753 <unknown>\n#8 0x560c06eb5baa <unknown>\n#9 0x560c06eb6dfe <unknown>\n#10 0x560c0734738b <unknown>\n#11 0x560c0734b307 <unknown>\n#12 0x560c07333e7c <unknown>\n#13 0x560c0734bec7 <unknown>\n#14 0x560c0731824f <unknown>\n#15 0x560c0736b2f8 <unknown>\n#16 0x560c0736b4c0 <unknown>\n#17 0x560c0737b646 <unknown>\n#18 0x7fdffac9ca94 <unknown>\n#19 0x7fdffad29c3c <unknown>","statusTrace":"self = <test_sign_in.TestFailedLogin object at 0x7ff9a258bb50>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"961ca78184069fed05fdfe6c907bd0e9\")>\nemail = 'teston110@mailinator.com', password = 'Test#12p@ssword'\n\n    @allure.title('TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation')\n    @pytest.mark.parametrize('email, password', LOGIN)\n    def test_03_02_01_to_03_02_10_login(self, driver, email, password):\n        \"\"\"Check Success and Failed Login, no error validation\"\"\"\n        sign_in_page = SignInPage(driver, SIGN_IN_URL)\n        sign_in_page.open()\n>       sign_in_page.fill_in_email_field(email)\n\ntests/test_sign_in.py:256: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sign_in_page.py:54: in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\npages/sign_in_page.py:41: in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\npages/base_page.py:80: in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"961ca78184069fed05fdfe6c907bd0e9\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7ff9a2bbd760>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x560c0737c7ca <unknown>\nE       #1 0x560c06e742f0 <unknown>\nE       #2 0x560c06ec4035 <unknown>\nE       #3 0x560c06ec4251 <unknown>\nE       #4 0x560c06f0a054 <unknown>\nE       #5 0x560c06ee89dd <unknown>\nE       #6 0x560c06f073b3 <unknown>\nE       #7 0x560c06ee8753 <unknown>\nE       #8 0x560c06eb5baa <unknown>\nE       #9 0x560c06eb6dfe <unknown>\nE       #10 0x560c0734738b <unknown>\nE       #11 0x560c0734b307 <unknown>\nE       #12 0x560c07333e7c <unknown>\nE       #13 0x560c0734bec7 <unknown>\nE       #14 0x560c0731824f <unknown>\nE       #15 0x560c0736b2f8 <unknown>\nE       #16 0x560c0736b4c0 <unknown>\nE       #17 0x560c0737b646 <unknown>\nE       #18 0x7fdffac9ca94 <unknown>\nE       #19 0x7fdffad29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1737855487853,"stop":1737855487915,"duration":62},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1737855487916,"stop":1737855487916,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1737857745255,"stop":1737857746507,"duration":1252},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Check Success and Failed Login, no error validation","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560c0737c7ca <unknown>\n#1 0x560c06e742f0 <unknown>\n#2 0x560c06ec4035 <unknown>\n#3 0x560c06ec4251 <unknown>\n#4 0x560c06f0a054 <unknown>\n#5 0x560c06ee89dd <unknown>\n#6 0x560c06f073b3 <unknown>\n#7 0x560c06ee8753 <unknown>\n#8 0x560c06eb5baa <unknown>\n#9 0x560c06eb6dfe <unknown>\n#10 0x560c0734738b <unknown>\n#11 0x560c0734b307 <unknown>\n#12 0x560c07333e7c <unknown>\n#13 0x560c0734bec7 <unknown>\n#14 0x560c0731824f <unknown>\n#15 0x560c0736b2f8 <unknown>\n#16 0x560c0736b4c0 <unknown>\n#17 0x560c0737b646 <unknown>\n#18 0x7fdffac9ca94 <unknown>\n#19 0x7fdffad29c3c <unknown>","statusTrace":"self = <test_sign_in.TestFailedLogin object at 0x7ff9a258bb50>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"961ca78184069fed05fdfe6c907bd0e9\")>\nemail = 'teston110@mailinator.com', password = 'Test#12p@ssword'\n\n    @allure.title('TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation')\n    @pytest.mark.parametrize('email, password', LOGIN)\n    def test_03_02_01_to_03_02_10_login(self, driver, email, password):\n        \"\"\"Check Success and Failed Login, no error validation\"\"\"\n        sign_in_page = SignInPage(driver, SIGN_IN_URL)\n        sign_in_page.open()\n>       sign_in_page.fill_in_email_field(email)\n\ntests/test_sign_in.py:256: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sign_in_page.py:54: in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\npages/sign_in_page.py:41: in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\npages/base_page.py:80: in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"961ca78184069fed05fdfe6c907bd0e9\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7ff9a2bbd760>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x560c0737c7ca <unknown>\nE       #1 0x560c06e742f0 <unknown>\nE       #2 0x560c06ec4035 <unknown>\nE       #3 0x560c06ec4251 <unknown>\nE       #4 0x560c06f0a054 <unknown>\nE       #5 0x560c06ee89dd <unknown>\nE       #6 0x560c06f073b3 <unknown>\nE       #7 0x560c06ee8753 <unknown>\nE       #8 0x560c06eb5baa <unknown>\nE       #9 0x560c06eb6dfe <unknown>\nE       #10 0x560c0734738b <unknown>\nE       #11 0x560c0734b307 <unknown>\nE       #12 0x560c07333e7c <unknown>\nE       #13 0x560c0734bec7 <unknown>\nE       #14 0x560c0731824f <unknown>\nE       #15 0x560c0736b2f8 <unknown>\nE       #16 0x560c0736b4c0 <unknown>\nE       #17 0x560c0737b646 <unknown>\nE       #18 0x7fdffac9ca94 <unknown>\nE       #19 0x7fdffad29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Open a browser","time":{"start":1737857746507,"stop":1737857746907,"duration":400},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Fill in Email field","time":{"start":1737857746907,"stop":1737857752024,"duration":5117},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560c0737c7ca <unknown>\n#1 0x560c06e742f0 <unknown>\n#2 0x560c06ec4035 <unknown>\n#3 0x560c06ec4251 <unknown>\n#4 0x560c06f0a054 <unknown>\n#5 0x560c06ee89dd <unknown>\n#6 0x560c06f073b3 <unknown>\n#7 0x560c06ee8753 <unknown>\n#8 0x560c06eb5baa <unknown>\n#9 0x560c06eb6dfe <unknown>\n#10 0x560c0734738b <unknown>\n#11 0x560c0734b307 <unknown>\n#12 0x560c07333e7c <unknown>\n#13 0x560c0734bec7 <unknown>\n#14 0x560c0731824f <unknown>\n#15 0x560c0736b2f8 <unknown>\n#16 0x560c0736b4c0 <unknown>\n#17 0x560c0737b646 <unknown>\n#18 0x7fdffac9ca94 <unknown>\n#19 0x7fdffad29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 54, in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 41, in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Check Email field is clickable","time":{"start":1737857746907,"stop":1737857752024,"duration":5117},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560c0737c7ca <unknown>\n#1 0x560c06e742f0 <unknown>\n#2 0x560c06ec4035 <unknown>\n#3 0x560c06ec4251 <unknown>\n#4 0x560c06f0a054 <unknown>\n#5 0x560c06ee89dd <unknown>\n#6 0x560c06f073b3 <unknown>\n#7 0x560c06ee8753 <unknown>\n#8 0x560c06eb5baa <unknown>\n#9 0x560c06eb6dfe <unknown>\n#10 0x560c0734738b <unknown>\n#11 0x560c0734b307 <unknown>\n#12 0x560c07333e7c <unknown>\n#13 0x560c0734bec7 <unknown>\n#14 0x560c0731824f <unknown>\n#15 0x560c0736b2f8 <unknown>\n#16 0x560c0736b4c0 <unknown>\n#17 0x560c0737b646 <unknown>\n#18 0x7fdffac9ca94 <unknown>\n#19 0x7fdffad29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 41, in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find clickable elements","time":{"start":1737857746907,"stop":1737857752023,"duration":5116},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560c0737c7ca <unknown>\n#1 0x560c06e742f0 <unknown>\n#2 0x560c06ec4035 <unknown>\n#3 0x560c06ec4251 <unknown>\n#4 0x560c06f0a054 <unknown>\n#5 0x560c06ee89dd <unknown>\n#6 0x560c06f073b3 <unknown>\n#7 0x560c06ee8753 <unknown>\n#8 0x560c06eb5baa <unknown>\n#9 0x560c06eb6dfe <unknown>\n#10 0x560c0734738b <unknown>\n#11 0x560c0734b307 <unknown>\n#12 0x560c07333e7c <unknown>\n#13 0x560c0734bec7 <unknown>\n#14 0x560c0731824f <unknown>\n#15 0x560c0736b2f8 <unknown>\n#16 0x560c0736b4c0 <unknown>\n#17 0x560c0737b646 <unknown>\n#18 0x7fdffac9ca94 <unknown>\n#19 0x7fdffad29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '#email')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"email","value":"'teston110@mailinator.com'"}],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"5db0d4e188113df","name":"Screenshot","source":"5db0d4e188113df.png","type":"image/png","size":25650},{"uid":"2e89d1174595c91f","name":"HTML source","source":"2e89d1174595c91f.html","type":"text/html","size":258925}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1737857752139,"stop":1737857752202,"duration":63},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1737857973906,"stop":1737857973906,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Failed Login Functionality"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_sign_in"},{"name":"subSuite","value":"TestFailedLogin"},{"name":"host","value":"fv-az1314-617"},{"name":"thread","value":"2774-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_sign_in"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'teston110@mailinator.com'"},{"name":"password","value":"'Test#12p@ssword'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"c3d6141023eb8bce.json","parameterValues":["'teston110@mailinator.com'","'Test#12p@ssword'"]}